client/view/Skip/Skip.js

```diff
@@ -0,0 +1,6 @@
+let skip = document.querySelector('[data-id=skip]')
+let form = document.querySelector('[data-id=query_text_area]')
```

Comment:
Let's make `const` like other `querySelector` :)

Reply:
I already move all them to `let`. `const` is only for `NAME_LIKE_THIS`

Reply:
OK :)
the main thing is to be uniform